pom.xml

```diff
@@ -107,17 +107,17 @@
     <antlr.version>3.5.2</antlr.version>
     <!-- Make sure to sync it with standalone-metastore/pom.xml -->
     <antlr4.version>4.9.3</antlr4.version>
-    <apache-directory-server.version>1.5.7</apache-directory-server.version>
+    <apache-directory-server.version>2.0.0-M1</apache-directory-server.version>
```

Comment:
IMO, exclusion of log4j:log4j:jar:1.2.16:test from apacheds-server-integ in service/pom.xml, standalone-metastore/pom.xml and itests/hive-minikdc/pom.xml would be better

Reply:
@Aggarwal-Raghav Thanks for pointing this out. Yes, this was bringing older log4j artifacts and making log4j version inconsistent across hive project. I have committed the change to exclude log4j artifacts from apache ds .